DEVELOPING.md

```diff
@@ -178,6 +178,39 @@ To run the server on an alternate port (e.g. 8001):
 tilt up --port=8001
 ```
 
+## Tilt Sharing
+
+Tilt has an experimental mode for sharing your Tilt view with other people. All sharing is public.
+
+To enable this feature, run:
+
+```
+tilt up --share=prod
```

Comment:
nice! This strikes me as a little weird tho -- is there a clean-ish way to make `--share` default to prod, tho allowing you specify `local` or `staging` if you want?

Reply:
good point! i split it up into two flags, --share and --share-mode